packages/web/src/app/shared/component-documentation/component-properties-table/table-base.ts

```diff
@@ -32,14 +32,25 @@ export class PropertyTableBaseDirective implements OnChanges {
         rows: this.getEventProps(this.props),
         description: 'Events let the component communicate with your app, notifying it of changes.',
       },
+      {
+        title: 'Functions',
+        expanded: true,
+        rows: this.getFunctionProps(this.props),
+        description:
+          'Functions can be imported separately from the component, and can sometimes be required for the component to work.',
+      },
     ];
```

Comment:
Note that long strings create overflow in the table header:

https://github.com/3lvia/designsystem/assets/54577949/a26319e8-7afd-42a0-b6ee-47dc8a66a4f0

Reply:
Should this be wrapped or just shorten the string?

Reply:
I'm unsure 🧐 Wrapping it would be best, but if I remember correctly that can be challenging in the table. Maybe you can ask @frideska? 😄